linux_os/guide/system/accounts/accounts-restrictions/password_storage/accounts_password_last_change_is_in_past/tests/user_password_change_time_in_future.fail.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+useradd testUserChage1
```

Comment:
nitpick: `testUserChage1` is used 3 times. Maybe a variable for it would improve the readability.

Reply:
:+1: changed in [135d663](https://github.com/ComplianceAsCode/content/pull/10031/commits/135d663c3dac8a18fb51088c0b758f095081c5fd)